plugins/web/opentelemetry-instrumentation-long-task/test/longTask.test.ts

```diff
@@ -101,4 +105,41 @@ describe('LongTaskInstrumentation', () => {
       "span duration should be longtask's"
     );
   });
+
+  it('should attach additional attributes from callback', async () => {
+    deregister();
+    const additionalAttributes = { foo: 'bar' };
+    longTaskInstrumentation = new LongTaskInstrumentation({
+      enabled: false,
+      observerCallback: () => additionalAttributes,
```

Comment:
This hook should be updated to the new signature (receive span as function argument and store attributes from the hook code)